paddle/fluid/framework/ir/mkldnn/activation_onednn_fuse_pass.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include "paddle/fluid/framework/op_desc.h"
+
+namespace paddle {
+namespace framework {
+namespace ir {
+
+static std::vector<std::string> GetSupportedActivations() {
```

Comment:
Optionally there could be `inline` added to all functions in this file.

Reply:
Done. I also removed `static` because there are no longer conflicts with multiple function definitions as previosuly, when _onednn_reuse.h_ was imported in multiple places.